pkgs/cloudflare-warp/default.nix

```diff
@@ -0,0 +1,114 @@
+{
+  stdenv,
+  lib,
+  autoPatchelfHook,
+  copyDesktopItems,
+  dbus,
+  dpkg,
+  fetchurl,
+  gtk3,
+  libpcap,
+  makeDesktopItem,
+  makeWrapper,
+  nftables,
+}:
+
+# Patched because of nixpkgs does not include latest and I would try this ver when addressing GH-749
+# https://github.com/NixOS/nixpkgs/blob/7de4869f8a4c21b1a2831df721e3d4599365ee69/pkgs/tools/networking/cloudflare-warp/default.nix
+stdenv.mkDerivation rec {
+  pname = "cloudflare-warp";
+  version = "2024.6.497";
```

Comment:
I may drop this PR if upstream merge https://github.com/NixOS/nixpkgs/pull/336116